app/views/charges/downgrade.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if current_user.has_role? :premium %>
+    <h4>Click to downgrade to Standard user!</h4>
+    <%= link_to "Downgrade", method: :downgrade, class: 'btn btn-danger', data: { confirm: 'Are you sure you want to downgrade to standard user and relinquish power?' } %>
```

Comment:
Brittany,
I meant to ask how to invoke the _downgrade_ method directly from application.html.erb. In you previous post,you did ask me to remove the get route and downgrade.html.erb, correct ?

Thanks,
Praveen.